samza-test/src/main/java/org/apache/samza/example/KeyValueStoreExample.java

```diff
@@ -68,7 +68,7 @@ public void describe(StreamApplicationDescriptor appDesc) {
         trackingSystem.getOutputDescriptor("pageViewEventPerMember",
             KVSerde.of(new StringSerde(), new JsonSerdeV2<>(StatsOutput.class)));
 
-    appDesc.withDefaultSystem(trackingSystem);
+    appDesc.withDefaultSystem(trackingSystem, null); // serde must be specified explicitly for intermediate streams
```

Comment:
When I initially read this comment, it seemed to conflict with the capability that the default serde could be set for intermediate streams. Does this comment refer to the fact that in this example, the serde is already specified explicitly for intermediate streams? If so, since this is an example that people might be looking at, do you think it might be good to clarify this?

Reply:
Alternatively, we can just say there is no default serde set for all intermediate streams.